src/Uno.Sdk/targets/Uno.Extensions.Implicit.Packages.ProjectSystem.targets

```diff
@@ -37,6 +37,13 @@
 
 	<ItemGroup Condition="$(UnoFeatures.Contains(';http;'))">
```

Comment:
Does Uno.Extensions.Http.Kiota reference Uno.Extensions.Http.WinUI? if not, we also need to include contains for refit/kiota.

Reply:
Actually it depends on Uno.extensions.http but not http.winui! Thanks :)